src/components/SearchResults.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/search-results.css";
+
+function SearchResults({ results }) {
+  if (!results.length) {
+    return (
+      <p className="no-images-message">Sorry, we couldn't find anything.</p>
```

Comment:
Looking at the rest of the classnames here, could this also be named using the atomic style to `search-results__message` ?